_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,242 @@
+  The same will be accomplished with `feefilter` messages by default after the
+  merge of [Bitcoin Core #19204][].
+
+- [Bitcoin Core #19133][] restores the bitcoin-cli `generate` command
+  which had been removed from Bitcoin Core. It is a shortcut for
+  `generatetoaddress(getnewaddress())` and is useful for testing with
+  regtest.
+
+- [Bitcoin Core #18027][] adds two options to the GUI's *File* menu for
+  working with Partially Signed Bitcoin Transactions ([PSBTs][topic
+  psbt]): *Load PSBT from file* and *Load PSBT from clipboard*.  When
+  one of those options is clicked and a PSBT is loaded, a dialog is
+  provided that allows the user to sign an incomplete PSBT if this
+  wallet has its key, broadcast a complete PSBT, or copy or save the
+  PSBT for processing with another tool such as [HWI][].  Combined with
+  other recent PSBT-related GUI changes (see Newsletters [#74][news74
+  psbtgui] and [#82][news82 psbtgui]) and [HWI's own GUI][hwi gui], this
+  makes it possible to use a PSBT-based process with Bitcoin Core for
+  the first time without using any RPCs.
+
+    ![Screenshot of PSBT dialog in Bitcoin Core GUI](/img/posts/2020-06-psbt-dialog.png)
+
+- [Bitcoin Core #16377][] updates the `walletcreatefundedpsbt` and
+  `fundrawtransaction` RPCs.  These RPCs normally use the wallet to
+  automatically choose which UTXOs to spend in an unsigned transaction,
+  but they also allow the user to specify one or more UTXOs they want to
+  spend in that transaction.  Previously, if the UTXOs selected by the
+  user weren't enough to pay for all of the transaction's outputs, the
+  wallet would automatically select more UTXOs to spend.  But, if a
+  user is manually selecting UTXOs, they may have some reason they don't
+  want to spend additional UTXOs, so the RPCs will now fail by default
+  if the user manually selects any UTXOs.  This default may be overridden
+  using the new `add_inputs` parameter to both RPCs.
+
+- [Eclair #1461][] adds several API endpoints that forward to Bitcoin
+  Core RPCs for relaying that program's wallet balances and other
+  information.  The goal is to make it easier to integrate Eclair with
+  the [Ride The Lightning][] node management dashboard.
+
+- [Bitcoin Core #19071][] adds documentation describing how developers
+  can contribute to the new and experimental [Bitcoin Core GUI
+  repository][].  Pull requests related to the GUI should be made into
+  this new repository, which will be bidirectionally synced with the
+  [main repository's master development branch][bitcoin/bitcoin] using
+  the *monotree* development model used by the Linux Kernel Project.
+  There are no direct user-visible changes from this split---users will
+  still receive the GUI in the official packaged versions of Bitcoin Core or
+  when building using `--with-gui` from the source code in the main
+  repository.
+
+    ![Illustration of monorepo versus monotree](/img/posts/2020-06-monorepo-vs-monotree.png)
```

Comment:
Using Master for the main repo is slightly confusing in this diagram, since the GUI fork will also have a master branch. Would changing Master to bitcoin/bitcoin and GUI to bitcoin-core/gui make this clearer?

Reply:
Good idea!  Done.

Reply:
The diagram is great. It would need a colored rather than transparent background to work with dark mode.

Reply:
We don't natively support a dark mode, so I think it's up to whatever feature or plugin you're using to invert or otherwise mutate images to make them work with dark mode.  If there are any HTML tags or CSS changes we can add to make that possible, I'm happy to do that, but I prefer to design most of my images so that they'll look good in print---which means assuming a white background.